pages/models.py

```diff
@@ -32,7 +32,8 @@
 
 # Set options for cmarkgfm for "unsafe" renderer, see https://github.com/theacodes/cmarkgfm#advanced-usage
 CMARKGFM_UNSAFE_OPTIONS = (
```

Comment:
Can we just move these options into the `unsafe_markdown_to_html` function now that they're not being shoveled into a lambda? I only made them a constant to make them more clear in the lambda definition.